sub/project_settings/{{ cookiecutter.__folder_name }}/backend/Makefile

```diff
@@ -142,3 +142,8 @@ acceptance-backend-start: ## Start backend acceptance server
 .PHONY: acceptance-image-build
 acceptance-image-build:  ## Build Docker Images
 	@DOCKER_BUILDKIT=1 docker build . -t $(IMAGE_NAME_PREFIX)-backend-acceptance:$(IMAGE_TAG) -f Dockerfile.acceptance --build-arg PLONE_VERSION=$(PLONE_VERSION)
+
+## Add bobtemplates features (check bobtemplates.plone's documentation to get the list of available features)
+add: $(BIN_FOLDER)/pipx
+	$(BIN_FOLDER)/pipx run plonecli add -b .mrbob.ini $(filter-out $@,$(MAKECMDGOALS))
```

Comment:
@erral We should add this also in https://github.com/plone/cookieplone-templates/blob/main/backend_addon/%7B%7B%20cookiecutter.__folder_name%20%7D%7D/Makefile -- otherwise it will be missing when the backend_addon template is used on its own and not part of the project template.

Reply:
Fixed in https://github.com/plone/cookieplone-templates/pull/63/commits/e3037c52b764f7af1782f82136829d6dd581d003